monitoring/micrometer-prometheus/src/main/java/io/quarkus/ts/micrometer/prometheus/PathPatternResource.java

```diff
@@ -0,0 +1,39 @@
+package io.quarkus.ts.micrometer.prometheus;
+
+import java.net.URI;
+
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.PathParam;
+import jakarta.ws.rs.core.Response;
+
+@Path("/test")
+public class PathPatternResource {
+    @GET
+    @Path("/redirect")
+    public Response redirect() {
+        String targetUrl = "https://quarkus.io/guides/";
+        return Response.status(Response.Status.FOUND) //302
+                .location(URI.create("/new-location"))
+                .build();
+    }
+
+    @GET
+    @Path("/not-found")
+    public Response notFound() {
+        return Response.status(Response.Status.NOT_FOUND).build(); //404
+    }
+
+    @GET
+    @Path("/moved/{id}")
+    public Response moved(@PathParam("id") String id) {
+        return Response.ok("Moved resource with ID: " + id).build(); // 200
```

Comment:
```suggestion
        return Response
                .status(Response.Status.MOVED_PERMANENTLY)
                .entity("Moved resource with ID: " + id)
                .build();
```
CI says you expect 301 but respond with 200